docs/ides/vscode.rst.include

```diff
@@ -10,13 +10,152 @@ It can be used together with any other coding language or be used to edit config
 
 It can be downloaded `here <https://code.visualstudio.com/download>`__.
 
+Once installed it is vital to ensure the Python interpreter is pointing to the *correct* Python executable.
+This depends on whether you are using :ref:`virtual environments <python-environments>`, see :ref:`here <environment-venv-vscode>`.
+Changing the interpreter location can be done by following `this instruction <https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter>`__.
+
+
+User Interface
+^^^^^^^^^^^^^^
+    .. image:: images/Interface.png
+      :width: 700
+      :align: center
+
+**Editor:** The main editor is where you write and edit your code or text files. It's the central area of VSCode where you'll spend most of your time.
+
+**Activity bar:** The sidebar, located on the left side of the interface, provides quick access to various functionalities and information about your project. 
+
+**Panel:** This is where you will find your terminal (currently open).
```

Comment:
I would add more context here, what about the different panes in the panel? By default a user will see many different things. OUTPUT etc. Clarifications on what they are, and what to be aware of could be useful?